index.html

```diff
@@ -312,12 +302,62 @@ <h3>
                 <li>
                     <a href="#"><img src="logo/angellist.png" alt=""></a>
                 </li>
+
             </ul>
         </div>
     </section>
+    </div>
+    <section class="popup_window">
+        <div class="popup">
+            <div class="mob_snap">
+                <img src="logo/mobile snap.png" alt="Mobile snapshoot">
+            </div>
+            <div class="desktop_snap">
+                <img src="logo/Desktop snap.png" alt="Desktop snapshoot">
+            </div>
+            <div class="popup-page">
+            <div class="popup_heading">
+                <h3>Keeping track of hundreds of components </h3>
+            </div>
+            <div class="popup_skills">
+                <ul class="popup_skills_mob">
+                    <li>Ruby on Rails</li>
+                    <li>CSS</li>
+                    <li>JavaScript</li>
+                </ul>
+                <ul class="popup_skills_desktop">
+                    <li>Codekit</li>
+                    <li>GitHub</li>
+                    <li>JavaScript</li>
+                    <li>Bootstrap</li>
+                    <li>Terminal</li>
+                    <li>Codepen</li>
+                </ul>
+            </div>
+            <div class="popup_description_Mob">
+                <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.</p>
 
-    <script src="menu.js"></script>
+                 <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.</p>
+
+            </div>
+            <div class="popup_description_Desktop">
+                <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea</p>
+            </div>
+            </div>
+            <div class="popup_btn">
+                <button>
+                    <a href="">See live</a>
+                    <img src="logo/Icon - Export.svg" alt="live icon">
+                </button>
+                <button>
+                    <a href="">See source</a>
+                    <img src="logo/Frame.png" alt="source frame">
+                </button>
+            </div>
+            <span class="close">&times;</span>
+        </div>
+    </section>
```

Comment:
- [ ] As the previous reviewer had mentioned, this popup section should be removed from the HTML file and created dynamically using javascript DOM methods. This is essential because it will make your code DRY, easy to maintain, and much easier to make additions. Modal popup data is populated with the data from the project array.

- [ ] Also your project section should be removed from the Html and generated dynamically using javascript and DOM methods. You will store all the projects as objects in an array. like so 👇🏾 
```ruby
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
``` 

Check an example of students' projects: https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95 and https://github.com/nicupop729/Portfolio/pull/11/commits/aabed37cc2208f52d0bd081a4cae8cdc83c49636.

Please fix all this and you will be fine.